### PR TITLE
Issue forms, pull request template.

### PR DESCRIPTION
**Pull request by Tech-TTGames**
## Description

I have migrated the normal .md issue forms to the GitHub .yaml issue forms to ensure ease of access for anyone using the reporting tools.

## Motivation and Context

I spent my time doing this becouse I forgot to commit the /dev branch changes from my F drive. So I was stuck with no access to the updated code.

## Changes Made

- Migrated .md issue templates to .yaml issue forms
- Separated Enhancement and Feature templates/forms.
- Added pull_request_template.md
- Added a question issue form.

## Issues Resolved

- N/A

## Testing Instructions

- N/A (No changes in code)

## Checklist

- [N/A] I have reviewed my own code and ensured it passes all linting tests.
- [N/A] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [N/A] I have added tests that prove my fix is effective or that my feature works.
- [X] I have run all tests and ensured they pass.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Discord Server
+    url: https:/discord.gg/google
```
Comment (kk5dire) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Do you really want to link to the r/Google discord for help with this?

**Review by kk5dire — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Tech-TTGames) [reply .github/ISSUE_TEMPLATE/config.yml:4]: For now? Why not. Later we may change it but atm the bot is literally just r/google.

Comment (benfoster04) [reply .github/ISSUE_TEMPLATE/config.yml:4]: We can add a different link as a method of contact, such as an email or a discord.bio site. This bot isn't strictly affiliated with r/Google, let's change it

**Review by benfoster04 — CHANGES_REQUESTED** [review]
Please see conversation regarding config.yml

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (kk5dire) [reply .github/ISSUE_TEMPLATE/config.yml:4]: > We can add a different link as a method of contact, such as an email or a discord.bio site. This bot isn't strictly affiliated with r/Google, let's change it

I agree @benfoster04, The bot isnt directly associated with r/Google (and for sure the raw code) and there is no reason any of the r/Google staff, or server members should be associated with it. The only bot we should assist with is the instance in the server. The only contact method for help with this code would be the issues pages, maybe we will work on something else later down the line, but that is the current case. I mirroed this fact in the security page, and in SUPPORT.md

**Comment by kk5dire** [comment]
Will approve merge when @benfoster04's CR is resolved.

**Review by benfoster04 — APPROVED** [review]
LGTM.
We will organise formal support contact links in due course.
Cleared for merge

**Pull request merged**